src/schemas/ui-view-schema.json

```diff
@@ -270,6 +270,53 @@
                       "description": "Expanded state of the timeline row",
                       "type": "boolean"
                     },
+                    "discreteOptions": {
```

Comment:
Need to discuss how we want to handle view migration. Haven't needed to do it in the past but now we may want to chat about it since moving the activityOptions into discreteOptions effectively breaks existing views that make use of activityOptions.

Reply:
Will be resolved by #1504